src/pages/Details.js

```diff
@@ -23,20 +35,25 @@ export default function Details({menusdata}) {
           <>
             <DesContainer>
               <Section>
-                <h2>Taste: </h2>
+                <H2>Taste: </H2>
                 {currentMenu.taste}
               </Section>
               <Section>
-                <h2>Spiciness:</h2>
-                {currentMenu.spiciness} / 5
+                <H2>Spiciness:</H2>
+
+                <div spiciness={currentMenu.spiciness}>
+                  {actives.map((active, index) => {
```

Comment:
Now here you can call the writen function and map over the returned array.

```suggestion
                  {calculatePeppers(spicinessValue).map((active, index) => {
```